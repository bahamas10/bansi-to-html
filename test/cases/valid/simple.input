[31mthis is red[0m
