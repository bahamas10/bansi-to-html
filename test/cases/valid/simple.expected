<span class="ansi-color-fg-default ansi-color-bg-default"></span><span class="ansi-color-fg-31 ansi-color-bg-default">this is red</span><span class="ansi-color-fg-default ansi-color-bg-default">
</span>
